### ISSUE TITLE
A11Y: Trap focus when modal is open

### ISSUE THREAD

**Issue by AnandChowdhary** [elninotech/uppload#61]
**Is your feature request related to a problem? Please describe.**
When the modal is open, a user is able to tab through the webpage in the background.

**Describe the solution you'd like**
Focus should be trapped within the Uppload modal, including the "close" button.

**Describe alternatives you've considered**
N/A

**Additional context**
This helps with https://uppload.js.org/a11y

**Comment by victorlap** [comment]
It's probably best to use something that is proven to work, like https://github.com/davidtheclark/focus-trap.
